fastlane/lanes/localization.rb

```diff
@@ -223,50 +266,80 @@
   #
   desc 'Downloads the localized metadata (for App Store Connect) from GlotPress for the WordPress app'
   lane :download_wordpress_localized_app_store_metadata do
-    # @FIXME: Replace this whole lane with a call to the future replacement of `gp_downloadmetadata` once it's implemented in the release-toolkit (see paaHJt-31O-p2).
-
-    # No need to `cd` into `fastlane` because of how Fastlane manages its paths internally.
-    sh './download_metadata.swift wordpress'
+    metadata_directory = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'metadata')
 
-    # @TODO: Make the `fastlane/metadata/en-US/release_notes.txt` path be the source of truth for the original copies in the future.
+    # FIXME: We should make the `fastlane/metadata/default/release_notes.txt` path be the source of truth for the original copies in the future.
```

Comment:
I'm gathering we're dropping the `@` prefix for `TODO` and `FIXME` comments?

I never looked into where the convention came from. I don't mind either way. Just asking to hopefully remember next time I find myself writing such notes 😄

Reply:
Yeah I think I have thought/assumed for a long time that the `@` prefix was expected and only recently discovered that rubocop and RubyMine suggest just `FIXME` and `TODO` without any `@` so I think it must have been just me adding that prefix for no reason after all 😅

Reply:
Refs:
 - https://docs.rubocop.org/rubocop/1.27/cops_style.html#stylecommentannotation
 - https://rubystyle.guide/#annotate-keywords